src/pages/a.js

```diff
@@ -26,6 +27,7 @@ const App = ({ isLoggedUser, user }) => {
           <PrivateRoute path="/" location={location}>
             <SchedulePage path="/summit-my-schedule" schedKey="my-schedule-main" location={location} headerTitle="My Schedule" />
             <ProfilePage path="/profile" isLoggedIn={isLoggedUser} user={user} location={location} />
+            <EditProfilePage path="/profile/edit" isLoggedIn={isLoggedUser} user={user} location={location} />
```

Comment:
review user, bc its should be user:userState and you were using user: userState.member